test/unit/inet-address-tests.js

```diff
@@ -97,19 +97,27 @@ describe('InetAddress', function () {
       helper.assertInstanceOf(val, InetAddress);
       assert.strictEqual(val.toString(), '10.11.12.13');
     });
-    it('should parse IPv6 Addresses with Embedded IPv4 Addresses', function () {
+    it('should parse IPv4-Mapped IPv6 addresses', function () {
       [
         ['0:0:0:0:0:FFFF:129.144.52.38',    '00000000000000000000ffff81903426'],
-        ['0:0:0:0:0:0:129.144.52.38',       '00000000000000000000000081903426'],
         ['::ffff:129.144.52.38',            '00000000000000000000ffff81903426'],
-        ['::ffff:254.255.52.32',            '00000000000000000000fffffeff3420'],
-        ['::13.1.68.3',                     '0000000000000000000000000d014403'],
-        ['fe::13.1.68.3',                   '00fe000000000000000000000d014403']
+        ['::ffff:254.255.52.32',            '00000000000000000000fffffeff3420']
       ].forEach(function (item) {
         var ip = InetAddress.fromString(item[0]);
         assert.strictEqual(ip.toString('hex'), item[1]);
       });
     });
+    it('should throw TypeError for invalid IPv6 address with embedded IPv4 address', function () {
+      [
+        ['0:0:0:0:0:0:129.144.52.38'],
```

Comment:
Shouldn't this one (`0:0:0:0:0:0:129.144.52.38`) be technically valid based on 2.5.5.1, or are we not supporting it because it is considered deprecated?